docs/analyzer/checker_and_analyzer_configuration.md

```diff
@@ -97,14 +88,24 @@ command with the `--z3-refutation` option.
 You can read more about refutation with the Z3 SMT Solver
 [here](https://docs.google.com/document/d/1-HEblH92VxdxDp04vDKjFa4_ZL9l2oPVLFtQUfLKSOo/).
 
-# Configure Clang tidy checkers
+# Configuring Clang tidy
+
+## Configuring the analyzer and checkers
+
+You can configure the clang-tidy analyzer and its checkers through CodeChecker with the `--analyzer-config` and the `--checker-config`
+flags of `CodeChecker analyze/check` commands as described in sections [Analyzer Configuration](#analyzer-configuration) and [Checker Configuration](#checker-configuration).
+
 
 ## Using Clang tidy configuration files
 
-__clang-tidy__ attempts to read configuration for each analyzed source file
+If you want to control the configuration of clang tidy from the `.clang-tidy` configuration files (instead of the CodeChecker command line)  you can use the `clang-tidy:take-config-from-directory=true` option. It will skip setting the checkers and checker configuration from CodeChecker (even if a profile was specified).
```

Comment:
In the above section you are using `clang-tidy` but here you are using `clang tidy` without dash. Please be consistent:
```suggestion
If you want to control the configuration of clang-tidy from the `.clang-tidy` configuration files (instead of the CodeChecker command line)  you can use the `clang-tidy:take-config-from-directory=true` option. It will skip setting the checkers and checker configuration from CodeChecker (even if a profile was specified).
```